examples/js/nodes/materials/PhongNode.js

```diff
@@ -131,7 +131,7 @@ THREE.PhongNode.prototype.build = function( builder ) {
 		var normal = this.normal ? this.normal.buildCode( builder, 'v3' ) : undefined;
 		var normalScale = this.normalScale && this.normal ? this.normalScale.buildCode( builder, 'fv1' ) : undefined;
```

Comment:
Hi @WestLangley. This would be a param equivalent of `texture2D( normalMap, vUv ).xyz` in  https://github.com/mrdoob/three.js/blob/dev/src/renderers/shaders/ShaderChunk/normalmap_pars_fragment.glsl#L20